verification_token/models.py

```diff
@@ -95,6 +103,12 @@ def check_key(self, key):
         """
         return self.is_valid and self.key == key
 
+    def set_extra_data(self, extra_data):
```

Comment:
teoreticky kdybys použil settery a gettery z properties https://www.programiz.com/python-programming/property#using a pole nazval dumped_extra_data nebo podobně, tak by to bylo jednodušší na použití

Reply:
To je otázka, na čtení bys použil dumped_extra_data, ale co na zápis? source_extra_data? Ty přidané prefixy mi přijdu spíše matoucí. Co se týče použití, je to skoro stejné, jen přistupuješ k atributu nebo metodě, ne?

Reply:
on je i setter úplně se stejnou funkčností jako getter a pak používáš A().a = 5 a nastaví ti to proměnnou _a, mrkni na ten odkaz

Reply:
ale je to jedno, tohle unesu :D